src/journal/cat.c

```diff
@@ -20,6 +20,7 @@
 
 static const char *arg_identifier = NULL;
 static int arg_priority = LOG_INFO;
+static int arg_stderr_priority = LOG_INFO;
```

Comment:
please set this to "-1" initially, as a marker of "not set yet", so that no additional bool for this is necessray